src/main/java/tr/com/srdc/cda2fhir/conf/Config.java

```diff
@@ -50,7 +52,7 @@
     
     public static final String NARRATIVE_PROPERTIES_FILE_PATH = "file:src/main/resources/narrative/customnarrative.properties";
 
-    private static boolean generateNarrative = true;
+    private static boolean generateNarrative = false;
     private static INarrativeGenerator narrativeGenerator;
 
     private static boolean generateDafProfileMetadata = true;
```

Comment:
the fhir resources are still generating with the DAF meta header, which we don't need either. I think treating this boolean the same as narrative should fix that, can you check it out?